<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelsSettings</class>
 <widget class="QWidget" name="labelsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Labels Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Minimum label score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minCountSpinBox">
       <property name="toolTip">
        <string>Show only labels that are rated at least that high</string>
       </property>
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maximum labels to show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numLabelsSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Show personal labels with score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="personalCountSpinBox">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="autoAddCheckBox">
     <property name="text">
      <string>Automatically add downloaded labels if none are assigned</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="minAutoAddCountLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Minimum label score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minAutoAddCountSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show only labels that are rated at least that high</string>
       </property>
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Blacklist</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Do not show the following labels:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KEditListBox" name="blacklistEditListBox">
        <property name="buttons">
         <set>KEditListBox::Add|KEditListBox::Remove</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>autoAddCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minAutoAddCountSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAddCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minAutoAddCountLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
